application/forms/PreferenceForm.php

```diff
@@ -208,8 +208,15 @@ public function createElements(array $formData)
                     )
                 );
             }
+        } else {
+            $globalThemeFile = Config::app()->get('themes', 'default');
+            $themeFile = StyleSheet::getThemeFile($globalThemeFile);
+            $file = $themeFile !== null ? @file_get_contents($themeFile) : false;
+            if ($file && strpos($file, StyleSheet::LIGHT_MODE_IDENTIFIER) === false) {
+                $disabled = ['', 'light', 'system'];
+            }
```

Comment:
This is nearly the same as the structure for the user's theme. Please combine both.

```
$theme = global theme or null;
if (user has chosen theme)
    $theme = user chosen theme

if ($theme !== null && $theme !== default theme)
    ...
```